applications: machine_learning: Add configuration for nRF54L15 DK (#18444)
nrfconnect/sdk-nrf: applications/machine_learning/app_desc.rst

  * ``tap`` - The device is tapped while placed on a flat surface.

  Unknown gestures, such as shaking the device, are recognized as anomalies.

> * The nRF52840 DK, nRF5340 DK and nRF54L15 all use the `nRF Connect SDK simulated sensor machine learning model`_.

Reviewer: ```suggestion
* The nRF52840, nRF5340 and nRF54L15 DKs use the `nRF Connect SDK simulated sensor machine learning model`_.
```

         * LED blinks with an average frequency if it is connected but is not actively forwarding data.
         * LED blinks rapidly if it is connected and is actively forwarding data.

   .. group-tab:: nRF54L15 DK

>       The nRF54L15 DK use monochromatic LEDs to display the application state.

Reviewer: ```suggestion
      The nRF54L15 DK uses monochromatic LEDs to display the application state.
```

>       The **LED1** displays the application state, and the **LED3** displays the signal generated by the simulated sensor.

Reviewer: ```suggestion
      The **LED1** displays the application state and the **LED3** displays the signal generated by the simulated sensor.
```

      * If the device is returning the machine learning prediction results, the **LED1** blinks for a predefined number of times and then turns off for a period of time.
        Then the sequence is repeated.
        The machine learning result is represented by the number of blinks:

>          * ``sine`` - 1 blink
>          * ``triangle`` - 2 blinks
>          * ``square`` - 3 blinks
>          * ``idle`` - 4 blinks

Reviewer: ```suggestion
         * ``sine`` - one blink
         * ``triangle`` - two blinks
         * ``square`` - three blinks
         * ``idle`` - four blinks
```

         If the machine learning model is running but has not detected anything yet or has detected an anomaly, the **LED1** is breathing.
      * If the device forwards data, the **LED1** has the following blinking patterns:

>          * LED blinks slowly if it is not connected.
>          * LED blinks with an average frequency if it is connected but is not actively forwarding data.
>          * LED blinks rapidly if it is connected and is actively forwarding data.

Reviewer: ```suggestion
         * Blinks slowly if it is not connected.
         * Blinks with an average frequency if it is connected but is not actively forwarding data.
         * Blinks rapidly if it is connected and is actively forwarding data.
```

Optionally, you can also connect to the device using `Edge Impulse's data forwarder`_ and forward data to `Edge Impulse studio`_ (after logging in).
See `Forwarding data to Edge Impulse studio`_ for details.

Testing with the nRF54L15 DK
---------------------------------------

After programming the application, perform the following steps to test the nRF Machine Learning application on the DK:

1. Turn on the development kit.
   The application starts in a mode that runs the machine learning model.

>    Initially, **LED3** displays the LED effect representing ``sine`` wave (1 blink), and **LED1** is breathing, because the signal was not yet recognized by the machine learning model.

Reviewer: ```suggestion
   Initially, **LED3** displays the LED effect representing ``sine`` wave (one blink), and **LED1** is breathing, because the signal was not yet recognized by the machine learning model.
```

   After a brief delay, the machine learning model recognizes the simulated signal.
   **LED1** and **LED3** display the same LED effect.
#. Press **Button 2** to change the generated acceleration signal.
   Right after the signal change, the effects displayed by LEDs are different.

>    After a brief delay, the machine learning model recognizes the ``triangle`` wave, and the same effect (2 blinks) is displayed by both LEDs.

Reviewer: ```suggestion
   After a brief delay, the machine learning model recognizes the ``triangle`` wave, and the same effect (two blinks) is displayed by both LEDs.
```

#. Press **Button 2** to again change the generated acceleration signal.

>    The ``square`` wave (3 blinks) is displayed only by the **LED3**.

Reviewer: ```suggestion
   The ``square`` wave (three blinks) is displayed only by the **LED3**.
```
